model_monitoring/components/tests/e2e/conftest.py

```diff
@@ -23,6 +25,22 @@
 lock_file = ".lock"
 
 
+def _get_subscription_id():
+    return os.environ.get("SUBSCRIPTION_ID", "ea4faa5b-5e44-4236-91f6-5483d5b17d14")
+
+
+def _get_tenant_id():
+    return os.environ.get("TENANT_ID", "72f988bf-86f1-41af-91ab-2d7cd011db47")
+
+
+def _get_resource_group():
+    return os.environ.get("RESOURCE_GROUP", "model-monitoring-canary-int-rg")
+
+
+def _get_workspace_name():
+    return os.environ.get("WORKSPACE_NAME", "model-monitoring-canary-int-ws")
```

Comment:
What's the purpose of providing defaults here? I'd rather not have at least the sub and tenant IDs in this repo.

Reply:
It's to run the tests from local!

Reply:
Would you please remove them? I know it adds one more step for local testing (maintaining a set of commands to set them before running pytest), but we shouldn't have them in the repo.